WordPressKit/PostServiceRemoteREST.m

```diff
@@ -306,6 +307,30 @@ - (void)restorePost:(RemotePost *)post
            }];
 }
 
+- (void)getLikesForID:(NSNumber *)postID
+              success:(void (^)(NSArray<RemoteUser *> * _Nonnull))success
+              failure:(void (^)(NSError * _Nullable))failure
+{
+    NSParameterAssert(postID);
+    
+    NSString *path = [NSString stringWithFormat:@"sites/%@/posts/%@/likes", self.siteID, postID];
+    NSString *requestUrl = [self pathForEndpoint:path
+                                     withVersion:ServiceRemoteWordPressComRESTApiVersion_1_1];
+    
+    [self.wordPressComRestApi GET:requestUrl
+                       parameters:nil
+                          success:^(id responseObject, NSHTTPURLResponse *httpResponse) {
+        NSArray *jsonUsers = responseObject[@"likes"];
+        if (success && jsonUsers) {
```

Comment:
It looks like nothing is returned if `jsonUsers` is not present. Shouldn't we just check for `success` and return an empty array of there are no likes?

Reply:
Nice catch! Will fix this in the next commit.